src/modules/library/word_models.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+from gensim.models import KeyedVectors, FastText
+
+
+class WordModels:
+    """
+    Class to load, train or save word embeddings.
+
+    Attributes:
+        embedding (gensim.models.keyedvectors.*): word embedding
+        model (type): word embedding model
+
+    Methods:
+        load(path, model_type='Word2vec'): loads word embedding model saved at 'path' and updates parameters 'embedding'
+            and 'model'. Argument 'model_type' describes which library we want to use.
+        train(documents, size=300, window=3, min_count=4, epochs=50): trains word embedding model on corpus 'documents'
+            using fastText.
+        save_to_file(file_name): saves the model to a file with path 'file_name'.
+
+    """
+
+    def __init__(self):
+        self.embedding = None
+        self.model = None
+
```

Comment:
I did the same in other modules too, but not for all attributes - only for the ones I thought this would be needed (for example not if an attribute is initialized with parameters of __init__ and then never changed).

I hope that's ok.